third_party/terraform/resources/resource_iam_audit_config.go

```diff
@@ -55,34 +55,6 @@ func ResourceIamAuditConfigWithBatching(parentSpecificSchema map[string]*schema.
 	}
 }
 
-func resourceIamAuditConfigCreate(newUpdaterFunc newResourceIamUpdaterFunc, enableBatching bool) schema.CreateFunc {
-	return func(d *schema.ResourceData, meta interface{}) error {
-		config := meta.(*Config)
-		updater, err := newUpdaterFunc(d, config)
-		if err != nil {
-			return err
-		}
-
-		ac := getResourceIamAuditConfig(d)
-		modifyF := func(ep *cloudresourcemanager.Policy) error {
-			ep.AuditConfigs = mergeAuditConfigs(append(ep.AuditConfigs, ac))
```

Comment:
Is `mergeAuditConfigs` still in use anywhere? As far as I can tell, the last user is it testing itself.